test/unit/image/pixels.js

```diff
@@ -42,6 +42,39 @@ suite('pixels', function() {
     test('works with integers', function() {
```

Comment:
```suggestion
     test('get(x,y) works with integers', function() {
```

Reply:
Hi @outofambit and @Spongman are you two okay with committing the two small test description changes so that we can move forward with merge? I don't want to let this and #3521 sit here for too long.